packages/next-swc/crates/next-core/src/next_import_map.rs

```diff
@@ -424,21 +406,18 @@ pub async fn insert_next_shared_aliases(
     import_map: &mut ImportMap,
     project_path: FileSystemPathVc,
     execution_context: ExecutionContextVc,
-    next_config: NextConfigVc,
 ) -> Result<()> {
     let package_root = next_js_fs().root();
 
-    if *next_config.mdx_rs().await? {
-        insert_alias_to_alternatives(
-            import_map,
-            mdx_import_source_file(),
-            vec![
-                request_to_import_mapping(project_path, "./mdx-components"),
-                request_to_import_mapping(project_path, "./src/mdx-components"),
-                external_request_to_import_mapping("@mdx-js/react"),
-            ],
-        );
-    }
+    insert_alias_to_alternatives(
```

Comment:
is it ok we don't reverse check mdx_rs flag:

`if !next_config.mdx_rs... { }`?

Reply:
resolved.